src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_WellKnownAttributes.cs

```diff
@@ -6447,38 +6553,50 @@ public int Prop
         get { return 1; }
         set { }
     }
+    [Obsolete(""Property"", true)]
```

Comment:
>[Obsolete(""Property"", true)] [](start = 4, length = 30)

is it intentional to have Obsolete on both the property and the setter here? The Obsolete on the setter does not seem to come into play